recipes/Hapi/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "Hapi" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/Hapi/archive/{{ version }}.tar.gz
+  sha256: 112e4e14562233b75f1157ceb8bb01c0ac33b4f469c28af8fcc1c2bf080e6775
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+
+  host:
+    - python >=2.7
```

Comment:
Conda-forge only builds >=3.6. No more legacy Python ;-p

Reply:
ok updated to 3.6

Reply:
However, in my previous recipe (oasis) in conda-forge, I considered python 2.7 and it went through